Userland/Libraries/LibWeb/Painting/FilterPainting.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2022, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibGfx/Filters/BrightnessFilter.h>
+#include <LibGfx/Filters/ContrastFilter.h>
+#include <LibGfx/Filters/GrayscaleFilter.h>
+#include <LibGfx/Filters/InvertFilter.h>
+#include <LibGfx/Filters/OpacityFilter.h>
+#include <LibGfx/Filters/SepiaFilter.h>
+#include <LibGfx/Filters/StackBlurFilter.h>
+#include <LibWeb/Layout/Node.h>
+#include <LibWeb/Painting/BorderRadiusCornerClipper.h>
+#include <LibWeb/Painting/FilterPainting.h>
+
+namespace Web::Painting {
+
+void apply_filter_list(Gfx::Bitmap& target_bitmap, Layout::Node const& node, Span<CSS::FilterFunction const> filter_list)
+{
+    for (auto& filter_function : filter_list) {
+        filter_function.visit(
+            [&](CSS::Filter::Blur const& blur) {
+                // Applies a Gaussian blur to the input image.
+                // The passed parameter defines the value of the standard deviation to the Gaussian function.
```

Comment:
Having the spec comments is great, but can we have a link to the spec too? Whether that's a link per function, or one to the whole list.